xarray/core/groupby.py

```diff
@@ -382,7 +382,15 @@ def fillna(self, value):
         Dataset.fillna
         DataArray.fillna
         """
-        return self._fillna(value)
+        def _yield_applied(this, other):
```

Comment:
I copped out and took your simple approach.  It's mainly because I'm not sure how to deal with more than 2 objects in `apply_ufunc` when `keep_attrs=True`.  Do I only take the first object's attributes?  Maybe best left for another PR.